core/app/[locale]/(default)/blog/[blogId]/page.tsx

```diff
@@ -47,49 +44,36 @@ export default async function Blog({ params }: Props) {
   }
 
   return (
-    <div className="mx-auto max-w-4xl">
-      <h1 className="mb-2 text-3xl font-black lg:text-5xl">{blogPost.name}</h1>
-
-      <div className="mb-8 flex">
-        <small className="mb-0 text-base text-gray-500">
-          {format.dateTime(new Date(blogPost.publishedDate.utc))}
-        </small>
-
-        {Boolean(blogPost.author) && (
-          <small className="text-base text-gray-500">, by {blogPost.author}</small>
-        )}
-      </div>
-
-      {blogPost.thumbnailImage ? (
-        <div className="mb-6 flex h-40 sm:h-80 lg:h-96">
-          <Image
-            alt={blogPost.thumbnailImage.altText}
-            className="h-full w-full object-cover object-center"
-            height={900}
-            src={blogPost.thumbnailImage.url}
-            width={900}
-          />
-        </div>
-      ) : (
-        <div className="mb-6 flex h-40 justify-between bg-primary/10 p-4 sm:h-80 lg:h-96">
-          <h3 className="mb-0 flex-none basis-1/2 self-start text-3xl font-bold text-primary">
-            {blogPost.name}
-          </h3>
-          <small className="mb-0 flex-none self-end text-xl font-bold text-primary">
-            {format.dateTime(new Date(blogPost.publishedDate.utc))}
-          </small>
-        </div>
-      )}
-
-      <div className="mb-10 text-base" dangerouslySetInnerHTML={{ __html: blogPost.htmlBody }} />
-      <div className="mb-10 flex">
-        {blogPost.tags.map((tag) => (
-          <Link className="me-3 block cursor-pointer" href={`/blog/tag/${tag}`} key={tag}>
-            <Tag content={tag} />
-          </Link>
-        ))}
-      </div>
-      <SharingLinks data={data} />
```

Comment:
Are we planning on handling sharing links in VIBES? Might want to track this just in case we do so it doesn't fall through the cracks